.buildkite/pipeline.yml

```diff
@@ -90,11 +57,10 @@ steps:
     depends_on:
      - test
      - rubocop
```

Comment:
You removed the `rubocop` step above (since it's run as part of Danger now)… but still depend on it here.

```suggestion
```